API_documentation_Python/Libraries_API_Python.ipynb

```diff
@@ -261,6 +265,155 @@
+    "        * permission: (sting) Permission type, can be 'read', 'write', 'admin', or 'owner'\n",
+    "        * public: (boolean) true means it is public\n",
+    "        * num_users: (integer) number of users with permissions to this library\n",
+    "        * owner: (string) identifier of the user who created the library\n",
+    "  * updates: (dict) Contains the following\n",
+    "        * num_updated: (integer) number of documents modified based on the response from solr\n",
+    "        * duplicates_removed: (integer) number of files removed because they are duplications\n",
+    "        * update_list: (list, dict) list of dictionaries such that a single element described the original bibcode (key) and the updated bibcode now being stored (item)\n",
+    "\n",
+    "The following type of user can read a library:\n",
+    "  - owner\n",
+    "  - admin\n",
+    "  - write\n",
+    "  - read"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'documents': ['1975CMaPh..43..199H', '1973PhRvD...7.2333B'],\n",
+       " 'solr': {'responseHeader': {'status': 0,\n",
+       "   'QTime': 34,\n",
+       "   'params': {'q': '*:*',\n",
+       "    'fl': 'bibcode,alternate_bibcode',\n",
+       "    'start': '0',\n",
+       "    'internal_logging_params': 'X-Amzn-Trace-Id=Root=1-64011d99-0652bc5e7903b9374dff1eee',\n",
+       "    'fq': '{!bitset}',\n",
+       "    'sort': 'date desc',\n",
+       "    'rows': '20',\n",
+       "    'wt': 'json'}},\n",
+       "  'response': {'numFound': 2,\n",
+       "   'start': 0,\n",
+       "   'docs': [{'bibcode': '1975CMaPh..43..199H'},\n",
+       "    {'bibcode': '1973PhRvD...7.2333B'}]}},\n",
+       " 'metadata': {'name': 'API Library',\n",
+       "  'id': 'Z7cCXDXQRC2HxB8CA8uu2g',\n",
+       "  'description': 'This library is created via API',\n",
+       "  'num_documents': 2,\n",
+       "  'date_created': '2023-03-02T22:05:01.292494',\n",
+       "  'date_last_modified': '2023-03-02T22:05:01.292502',\n",
+       "  'permission': 'owner',\n",
+       "  'public': True,\n",
+       "  'num_users': 1,\n",
+       "  'owner': 'kelly.lockhart'},\n",
+       " 'updates': {'num_updated': 0, 'duplicates_removed': 0, 'update_list': []}}"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# get the data for the library created above\n",
+    "results = requests.get(\"https://api.adsabs.harvard.edu/v1/biblib/libraries/Z7cCXDXQRC2HxB8CA8uu2g\",\n",
+    "                       headers={'Authorization': 'Bearer ' + token})\n",
+    "results.json()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "using the `raw` query parameter allows the user to return the exact bibcodes present in the library, regardless of whether or not they are present in ADS. "
```

Comment:
Capitalize this - it sort of blends into the request response above when rendered.

Reply:
Capitalized and added to the unix notebook